app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Service catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization, both {{site.base_gateway}} services and external integration resources. This catalog is the single source of truth for your organization’s service inventory and their dependencies.
+
+Specifically, Service Catalog addresses the following problems:
+
+* **Service discovery:** Gain visibility into all your services, including unrecognized or undiscovered APIs that live in your organization’s fragmented ecosystems. 
+* **Service ownership:** Keep track of your organization's service ownership by mapping teams and services. 
+* **Service health:** Consolidate key service health metrics into a single view, so you no longer need to check multiple tools (like GitHub and PagerDuty) for recent activity and status. It also helps you quickly identify and reuse existing services, reducing duplication and effort.
```

Comment:
```suggestion
* **Service health:** Consolidate key service health metrics into a single dashboard, allowing you to interact with other tools from one place. 
```
"It also helps you quickly identify and reuse existing services, reducing duplication and effort."
How is it helping you do that?